src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Command/ScaleCommand.php

```diff
@@ -19,21 +19,83 @@ class ScaleCommand implements CommandInterface
      */
     public function execute(&$image, $parameters)
```

Comment:
Can you please add a type hint to `$image`? It's not very easy to see what kind of object is handled here.